Library/Services/Implementation/NfieldSurveySampleService.cs

```diff
@@ -64,15 +64,17 @@ public Task<int> DeleteAsync(string surveyId, string respondentKey)
             Ensure.ArgumentNotNullOrEmptyString(surveyId, nameof(surveyId));
             Ensure.ArgumentNotNullOrEmptyString(respondentKey, nameof(respondentKey));
 
-            var uri = SurveySampleUrl(surveyId);
+            var uri = SurveySampleUrl(surveyId) + @"/Delete";
             var filters = new List<SampleFilter>
             {
                 new SampleFilter{Name = "RespondentKey", Op = "eq", Value = respondentKey}
             };
 
-            return Client.DeleteAsJsonAsync<IEnumerable<SampleFilter>>(uri, filters)
+            return Client.PutAsJsonAsync<IEnumerable<SampleFilter>>(uri, filters)
```

Comment:
Are you sure it is a put now? I looked at the changes in the api and I couldn't see the change to make it a put anywhere... Am I missing something?

Reply:
You are right, in the API this should be a DELETE and not a PUT.